### PR TITLE
Feature/get cwl logs

### PR DESCRIPTION
**Pull request by raar1**
Gets the logs from cwltoil, and stores them in an ```ROCrate```'s ```run_log``` var.

### CODE REVIEW

File: fairworkflows/cwl.py [hunk 1]
```diff
@@ -94,14 +100,19 @@ def run_workflow(wf_path: Union[Path, str], inputs: Dict[str, any], output_dir:
...
     except SystemExit as e:
         raise CWLException(f'Workflow {wf_path} with inputs {inputs} has failed.', e)
 
     _logger.debug('CWL tool has run successfully.')
 
-    return output_dir
+    with open(log_path, 'r') as infile:
+        runlogs = infile.read()
+
+    return runlogs
```
Comment (dsmits) [inline fairworkflows/cwl.py:115]: I don't think we necessarily need to add this part, I'm assuming you put this here for debugging. At some point we need to have a discussion about what we need to have as output of a workflow run. A dict listing all output files for example.
I'm not sure if we need to have the log as output as well, or that we should keep that information away from the user.

Comment (dsmits) [reply fairworkflows/cwl.py:115]: Hm I see now that you need this log as output to store in the RO crate. I suggest we return a tuple for now: `(OUTPUT_DIR, LOG)`

File: fairworkflows/cwl.py [hunk 2]
```diff
@@ -80,31 +82,39 @@ def run_workflow(wf_path: Union[Path, str], inputs: Dict[str, any], output_dir:
     _logger.debug(f'Running CWL tool at {wf_path}')
     _logger.debug(f'Input values: {inputs}')
     _logger.debug(f'Results will be written to {output_dir}')
+
+    if output_dir is None:
+        output_dir = Path(tempfile.mkdtemp())
+    log_path = str(output_dir / 'log.txt')
     output_dir = str(output_dir)
+
+    print('AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA', output_dir, log_path)
```
[the remark this answers is not in the document]
Comment (raar1) [reply fairworkflows/cwl.py:89]: Yes, sorry :( It was the state of mind I was in when I was doing this...

File: fairworkflows/cwl.py (hunk 1 above)
Comment (raar1) [reply fairworkflows/cwl.py:115]: OK, no problem, I'll make that the returned values

**Pull request merged**